resources/lang/en/events/injury/desert.json

```
@@ -1670,7 +1682,9 @@
             "desert"
         ],
         "season": [
-            "leaf-fall", "newleaf", "greenleaf"
+            "leaf-fall",
+            "newleaf",
+            "greenleaf"
         ],
         "weight": 20,
         "event_text": "m_c sneaked out of camp and was found, confused and disoriented, in the full noon sun.",
```

Ik this wasn't one you were reviewing but I do think it flows oddly. Also hatred of the word "sneaked", my british ass wants to say it should be snuck but I don't know if that's true in American English...

```suggestion
        "event_text": "m_c sneaked out of camp and was later found wandering in the full noon sun, confused and disoriented.",
```

---

I also hate the word sneaked lmao, but it's technically more correct? i think? but i will gladly destroy it